Gemfile.lock

```diff
@@ -158,7 +158,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  govuk_tech_docs
+  govuk_tech_docs (~> 1.3)
```

Comment:
not sure what's going on here but `Gemfile.lock` should not change unless `Gemfile` is changed and you re-run bundle install.

Reply:
I think this must be because I re-ran `bundle install` after the recent updates to the tech-docs-template

Reply:
I also pinned this version in `Gemfile` (now in master). I have the feeling it was that change I made that was half way and didn't update the `Gemfile.lock` accordingly.